pkg/terra/client/client.go

```diff
@@ -444,3 +442,8 @@ func (c *Client) Balance(addr sdk.AccAddress, denom string) (*sdk.Coin, error) {
 	}
 	return b.Balance, nil
 }
+
+//TODO how much of app.MakeEncodingConfig to reproduce?
```

Comment:
`app.MakeEncodingConfig()` is documented for testing only and calls global `Register*` funcs, so we shouldn't use it in production and we can't call it more than once anyways. `params.MakeEncodingConfig` swaps right in, but I don't know if we depend on the global register calls too (and would need to reproduce calling those once from an init).

Reply:
Test failures suggest we need those, so added to a `func init()`.